integration/integration.cabal

```diff
@@ -51,7 +51,6 @@ common common-all
     MultiWayIf
     NamedFieldPuns
     NoImplicitPrelude
-    NoImportQualifiedPost
```

Comment:
tripped over this as well yesterday.  :)  please don't remove this.  and maybe add a `-- (sic!)` so it won't be almost removed a third time?

https://github.com/wireapp/wire-server/pull/3793

Reply:
alright :)